vhs-frontend/src/pages/VHSDetails.tsx

```diff
@@ -23,7 +23,7 @@ export const VHSDetails = () => {
       setVhsInfo(response.data);
     } catch (err) {
       console.log(err);
-      notFoundLink("/notfound");
+      notFoundLink('/notfound');
```

Comment:
The `notFoundLink` method should be called `navigate`. Change its name in the line where you use `useNavigate()` because this better reflects what the returned hook method does.

Also, change the notfound route to be `not-found`, we usually split words in urls with a dash.